ui/src/style/s2-link.less

```diff
@@ -0,0 +1,20 @@
+/**
+ * Styles for the <S2Link /> component.
+ */
+
+a[href].s2-link {
+  &,
```

Comment:
what does this empty ampersand do?

Reply:
Ah, it's `less`'s "parent" selector:

http://lesscss.org/features/#parent-selectors-feature

So, this will evaluate to:

```css
a[href].s2-link,
a[href].s2-link:hover,
a[href].s2-link:visited {
   // css
}
```

Reply:
Cool! Thanks for the just-in-time learnings ;-)